psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -356,16 +354,12 @@ void DFHelper::prepare_sparsity() {
     }
     small_skips_[nbf_] = coltots;
 
-    // build indexing skips for sparse, symmetric pQq integrals
-    // symm_big_skips: outer indexing jumps. for the p index
-    // symm_ignored_columns: number of columns that should be ignored, for a given p index,
-    //                              due to triangular symmetry
-    // symm_small_skips: size of q for each p index, technically small_skips[p] - symm_ignored_columns[p]
+    // ==> Symmetric indexing. <==
```

Comment:
`p`, `q`, `Q` are widespread in notation. really want to remove them, even with fuller docs below?

Reply:
I can see that as a justification for using `p`, `q` and `Q` in the fuller docs in the header, but I'm less clear why that means these comments should stay. My concern is that the header should be the single source of truth for comments about what a variable means.

Reply:
I definitely support using pqQ in header docs. I suspect some folks won't go to the header and would instead grasp appreciatively at existing comment docs, but I appreciate the single-source argument, too.

Reply:
I moved the p/q information into the comments in the header, but I want to stay firm on "variable explanations are in the header."